guides/shop-managers-guide/72-payments.md

```diff
@@ -0,0 +1,5 @@
+# Payments
+
+To setup Payments go to the admin and select Settings in the side bar and select Payments. You will see a list of payment methods with a button to enable or disable that payment option. The enabled payment methods are visible to the customer at checkout. 
```

Comment:
I think we should call out that the example payment method is good for testing before setting up Stripe since a lot of people won't be using Stripe.